### DISCUSSION TITLE
Does Opensearch operator version 2.4.0 supports the Opensearch version 2.10.0?

### DISCUSSION THREAD

**Discussion by Gokul-Radhakrishnan** [Opster/opensearch-k8s-operator#630] [Q&A] [answered]
Does Opensearch operator version 2.4.0 supports the Opensearch version 2.10.0?

I don't see the compatibility table updated with operator version 2.4.0

<img width="875" alt="image" src="https://github.com/Opster/opensearch-k8s-operator/assets/38651133/4850b2b1-01a8-4df9-8c23-54668cfcdfc2">

**Comment by swoehrl-mw** [comment, marked as answer]
Hi @Gokul-Radhakrishnan. We have not explicitly tested Opensearch 2.10 with the operator yet, but there is nothing in the releases notes that hint at imcompatibilities. As such you can just try it out, the operator does not prevent you from using newer versions. The table is more along the lines of someone has used that version combination successfully.
